In [4]:
import numpy as np
def rule1(c, errA):
    errQ = errA * np.abs(c)
    return errQ

def rule2(c, m, a, errA):
    errQ = np.abs(c * m * a ^ (m - 1)) * errA
    return errQ
    
def rule3(errA, errB):
    errQ = np.sqrt(errA**2 + errB**2)
    return errQ

def rule4(q, a, errA, m, b, errB, n):
    errQ = np.abs(q) * np.sqrt((m*(errA/a))**2 + (n*(errB/b))**2)
    return errQ

print(rule4(.3199, .0859, .0001, .09434, 1, 1, 0))

3.513313853317811e-05


In [8]:
import numpy as np

x = np.array([1.1, 1.3, 1.4, 0.9, .95, 1.05])
print(np.average(x))
print(np.std(x))
print((np.std(x)/np.sqrt(len(x))))

1.1166666666666667
0.17950549357115014
0.073282810879294


The mean of the dataset is 1.117, the standard deviation is 0.179 and the error in the mean is 0.0733

$\delta Q = \sqrt{(\frac{\delta A}{A})^2 + (\frac{\delta B}{B})^2)}$